flink-dist/src/main/flink-bin/bin/flink-console.sh

```diff
@@ -116,4 +117,5 @@ echo $$ >> "$pid" 2>/dev/null
 # Evaluate user options for local variable expansion
 FLINK_ENV_JAVA_OPTS=$(eval echo ${FLINK_ENV_JAVA_OPTS})
 
+exec 1>"${out}"
```

Comment:
Given that we are using `log4j-console.properties` for the log configuration, all the Flink logs will also be mirrored to the stdout. Then we will have duplicated logs for `*.log` and `*.out`.

Actually, we already have some discussion about the `.out` file in this PR https://github.com/apache/flink/pull/11839.

Reply:
The file size of redirection is a concern from community. It makes sense to me. The lack of out file from web was reported by our internal users who use stream.print() to verify some execution result. Given stream.print() is an supported functionality, l feel it is a kind of lacking integrity if flink on k8s behaves differently with flink on yarn. Shall we revisit the problem together with Till and Zentol?